tests/test_sync.py

```diff
@@ -65,6 +65,23 @@ def test_ApiSync_stop_calls_quit(mocker, session_maker, homedir):
     api_sync.sync_thread.quit.assert_called_once_with()
 
 
+def test_ApiSync_sync_starts_now(mocker, session_maker, homedir):
+    """
```

Comment:
now that I think about it, we can probably just remove this test and the ApiSync.sync() (immediate sync) method since we no longer use it when source or conversation is deleted